src/main/java/org/opensearch/flowframework/transport/ReprovisionWorkflowTransportAction.java

```diff
@@ -290,7 +306,7 @@ private void executeWorkflow(Template template, List<ProcessNode> workflowSequen
                 List<ProcessNode> predecessors = processNode.predecessors();
                 logger.info(
                     "Queueing process [{}].{}",
-                    processNode.id(),
+                    String.format("%s (type: %s)", processNode.id(), processNode.workflowStep().getName()),
```

Comment:
You shouldn't need to use `String.format()` with logger substitutions.  This should be achievable with something like `"Queueing Process [{} (type: {})].()"`